README.md

```diff
@@ -787,6 +787,54 @@ List of breaking changes from v2 to v3:
   - v2: `-sp` to v3: `-p`
 - Project naming convention while checking out a project is now - `OWNER_PROJECT-NAME`. For example if we have `owner/some-name` the project will be checked out as `owner_some-name`. In v2 this would have been checked out as `owner_some_name`
 
+## Found a regression?  
```

Comment:
can you please add the section to the initial guidelines

Reply:
initial guidelines as in the table of content?

Reply:
indeed @shubhbapna